services/api_service/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.9.6-alpine
+
+WORKDIR /usr/src/api_service
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN adduser -D user
+USER user
+
+RUN pip3 install --upgrade pip
+COPY ./requirements.txt .
+RUN pip3 install -r requirements.txt
+
+COPY . .
```

Comment:
If you move this line up, you won't need to copy *requirements.txt* separately.

Reply:
but copy do not create new image. I believe we can leave as it is. For me it's more readable